.github/workflows/towerfulltest_hifi.yml

```diff
@@ -0,0 +1,27 @@
+name: Tower AWS full size tests HiFi
```

Comment:
So much repeat code :/ I wonder if you could parameterize these.

Reply:
Ya i felt that when I was writing this. I looked at nf-core to see if other people ran multiple AWS full tests to see if there was a better way to structure this and I couldn't find any examples. I'm a relative newbie with github actions/workflows, do you have any ideas of how these could be combined?

Reply:
Okay, so you can do this basically: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action.

But seems annoying and we can potentially get to it later. Doesn't seem super high priority.

Reply:
ok posted as issue #34 and will deal with it in the future!